experiments/interpretation/dutchf3_patch/local/test.py

```diff
@@ -268,6 +317,19 @@ def _evaluate_split(
         logger.info("Running in Debug/Test mode")
         test_loader = take(1, test_loader)
```

Comment:
I think for debug mode its better to avoid `toolz.take` and use PyTorch's built-in `torch.utils.data.Subset` instead, what do you think? toolz.take caused me a lot of problems when i tried to use it..

Reply:
discussed and resolved with data.Subset attempt (until it breaks - then revert back to toolz.take)